src/test/java/com/google/api/codegen/testdata/discoveries/py/py_logging.v2beta1.json.baseline

```diff
@@ -27,18 +30,19 @@ credentials = GoogleCredentials.get_application_default()
 service = discovery.build('logging', 'v2beta1', credentials=credentials)
 
 
-
 list_log_entries_request_body = {
-# TODO: Add desired entries of the 'list_log_entries_request_body' dict
+    # TODO: Add the desired entries.
 }
 
+
 entries = service.entries()
 request = entries.list(body=list_log_entries_request_body)
 while request is not None:
     response = request.execute()
 
     for log_entry in response['entries']:
-        # TODO: Add code here to process each 'log_entry' resource
+        # TODO: Use log_entry:
+        pprint(log_entry)
```

Comment:
You can just say `pass`, rather than suggesting that `pprint()` needs to be done. See how we did this for VKit:

http://pythonhosted.org/gax-google-pubsub-v1/google.pubsub.v1.publisher_api.html#google.pubsub.v1.publisher_api.PublisherApi.list_topics